hook/index.rb

```diff
@@ -13,6 +13,7 @@
   Docker::Container.expire_cache!
   me = Docker::Container.me
   containers = Docker::Container.all - [me]
+  containers.sort_by! { |c| c.name }
```

Comment:
Ah, Sorry there are two loops 🙈 

Could you move this line to L18?
`containers` instance is assigned by L6 only if all networks is connected to yaichi's container.